src/sentry/snuba/discover.py

```diff
@@ -597,12 +597,14 @@ def get_facets(
 
     fetch_projects = False
     if len(params.get("project_id", [])) > 1:
+        # TODO(nar): Since we pop a result to fill it with project data, we need to account for this offset
```

Comment:
Going to follow this up in another PR. I need to validate the pagination results in prod first and unblock myself for the frontend